src/Fixer/Import/GroupImportFixer.php

```diff
@@ -188,7 +188,12 @@ private function addGroupUseStatements(array $statements, Tokens $tokens)
      */
     private function getNamespaceNameWithSlash(NamespaceUseAnalysis $useDeclaration)
     {
-        return substr($useDeclaration->getFullName(), 0, strripos($useDeclaration->getFullName(), '\\') + 1);
+        $position = strripos($useDeclaration->getFullName(), '\\');
```

Comment:
```suggestion
        $position = strrpos($useDeclaration->getFullName(), '\\');
```
We don't need case insensitive check here.